lib/asciidoctor/gb/fonts_manifest.yaml

```diff
@@ -0,0 +1,6 @@
+---
+SimSun:
+Cambria:
+SimHei:
```

Comment:
Need to verify these font names in https://github.com/fontist/formulas

Reply:
@ronaldtse https://github.com/fontist/formulas/issues/58 - reported